modules/orientdb/src/main/java/org/testcontainers/containers/OrientDBContainer.java

```diff
@@ -68,12 +69,15 @@ public OrientDBContainer(final DockerImageName dockerImageName) {
         serverPassword = DEFAULT_SERVER_PASSWORD;
         databaseName = DEFAULT_DATABASE_NAME;
 
+        LogMessageWaitStrategy waitForLogs = new LogMessageWaitStrategy()
+            .withRegEx(".*Gremlin started correctly.*");
+
         WaitStrategy waitForHttp = new HttpWaitStrategy()
             .forPort(DEFAULT_HTTP_PORT)
             .forStatusCodeMatching(response -> response == HTTP_OK);
 
         waitStrategy = new WaitAllStrategy()
-            .withStrategy(Wait.forListeningPort())
+            .withStrategy(waitForLogs)
             .withStrategy(waitForHttp)
```

Comment:
I would suggest we remove `WaitAllStrategy` and the `HttpWaitStrategy` in this case and _only_ use the `LogMessageWaitStrategy`. Or why did you feel like keeping the HTTP wait?

Reply:
Yeah, I was very much on the fence about this, and my first draft removed the http wait as well. My (weak) logic: we have the http wait code already, and keeping it would guard against unforeseen breakage.

However, maybe the consequences of this http wait failing are, really, not much different to letting the test fail. So I'll remove 😄

Reply:
>  we have the http wait code already, and keeping it would guard against unforeseen breakage.

This sounds reasonable, but since we are trying to increase stability while being quite uninformed about the actual and intended behavior of the image, I'd argue it's better to trim down code to what we understand, instead of adding more duct-tape 🦆 .

Reply:
Agreed, have removed.